core/src/main/java/hudson/AboutJenkins.java

```diff
@@ -10,7 +12,7 @@
  * 
  * @author Kohsuke Kawaguchi
  */
-@Extension
+@Extension @Symbol("about")
```

Comment:
As an illustration of the use case, I want the system-config-dsl plugin or something similar to be able to filter out extensions for OEM use cases.